common.sh

```diff
@@ -75,6 +75,8 @@ export IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metalkube/metalkube-ironic"}
 export IRONIC_INSPECTOR_IMAGE=${IRONIC_INSPECTOR_IMAGE:-"quay.io/metalkube/metalkube-ironic-inspector"}
 export IRONIC_DATA_DIR="$WORKING_DIR/ironic"
 
+export KUBECONFIG="ocp/auth/kubeconfig"
```

Comment:
Would it be better to make this an absolute path with $SCRIPTDIR?  Then it will work even if scripts end up in subdirectories, and also if folks want to source common.sh in their profile or other test scripts, they can?

Reply:
Fixed, thanks!